api/apps/api/src/modules/scenario-specification/__tests__/create-candidate/fixtures.ts

```diff
@@ -0,0 +1,48 @@
+import { v4 } from 'uuid';
+import {
+  FeatureConfigInput,
+  ScenarioSpecification,
+} from '../../scenario-specification';
+import { SpecificationOperation } from '../../specification-operation';
+import { CandidateSpecCreated } from '../../candidate-spec-created.event';
+
+export const getFixtures = () => {
+  const scenarioId = v4();
+  const baseFeatureId = v4();
+  const stratificationFeatureId = v4();
+
+  const scenarioSpec = new ScenarioSpecification(scenarioId);
+
+  const splitConfig: FeatureConfigInput = {
+    operation: SpecificationOperation.Split,
+    baseFeatureId: baseFeatureId,
+  };
+
+  const stratificationConfig: FeatureConfigInput = {
+    operation: SpecificationOperation.Stratification,
+    baseFeatureId: baseFeatureId,
+    againstFeatureId: stratificationFeatureId,
+  };
+
+  return {
+    scenarioSpec,
+    WhenSubmittingDraft: () => {
+      scenarioSpec.createCandidate([splitConfig, stratificationConfig], true);
+    },
```

Comment:
WhenCreatingCandidateWithTwoConfigurations

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/371/commits/4dbad00ade491ccca947032a295834751eab9b2d